tools/wt_binary_decode.py

```diff
@@ -319,6 +396,11 @@ def unpack_uint64(b):
     val, arr = unpack_int(arr)
     return val
 
+def discard_unpacked_uint64(b):
```

Comment:
I think we should change this to something like:
```
def unpack_uint64_with_sz(b):
    start = b.tell()
    arr = file_as_array(b)
    val, arr = unpack_int(arr)
    sz = b.tell() - start
    return (val, sz)
```
Then in print_timestamps we should call this like:

```
        if extra & WT_CELL_TS_START != 0:
            t, sz = unpack_uint64_with_sz(b)
            pagestats.start_ts_sz += sz
            pagestats.num_start_ts += 1
            p.rint(' start ts: ' + ts(t))
```

Reply:
Done